.gitignore

```diff
@@ -0,0 +1,138 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Woah, this is one of the biggest .gitignore files I've ever seen. Do we really need all of this?